src/scales/scale.linear.js

```diff
@@ -156,7 +158,8 @@ module.exports = function(Chart) {
 			}
 		},
 		getLabelForIndex: function(index, datasetIndex) {
-			return +this.getRightValue(this.chart.data.datasets[datasetIndex].data[index]);
+			var value = this.chart.data.datasets[datasetIndex].data[index];
```

Comment:
inline this instead of creating temporary `value` variable

Reply:
done